static/app/views/performance/traceDetails/transactionBar.tsx

```diff
@@ -594,8 +597,7 @@ class TransactionBar extends Component<Props, State> {
           style={{
             width: `calc(${toPercent(1 - dividerPosition)} - 0.5px)`,
             paddingTop: 0,
-            alignItems: isTraceError(transaction) ? 'normal' : 'center',
```

Comment:
No more align?

Reply:
<img width="506" alt="Screenshot 2023-08-15 at 5 00 40 PM" src="https://github.com/getsentry/sentry/assets/60121741/be69a035-3a3f-4357-b5c2-960b38ea347e">

The styled component RowCell already has alignment I was over writing it initially. The negative positioning renders the flames at the correct position.